src/app/topics/topics.page.html

```diff
@@ -51,6 +51,13 @@
         <app-topic-6 *ngIf="topic.number===6"></app-topic-6>
         <app-topic-7 *ngIf="topic.number===7"></app-topic-7>
         <app-topic-8 *ngIf="topic.number===8"></app-topic-8>
+        <app-topic-8 *ngIf="topic.number===9"></app-topic-8>
```

Comment:
This needs to change to
`<app-topic-9 *ngIf="topic.number===9"></app-topic-9>`
and same for 10 through 15.

Reply:
Omg, thank you! Am on it asap

Reply:
@michaelclapham  I have now got the numbers right.

Reply:
Looks good